apps/web/src/server/views/applications/representations/representation-details/task-log.njk

```diff
@@ -0,0 +1,67 @@
+{% extends "applications/layouts/applications.layout.njk" %}
+
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "./components/representation-status-tag.njk" import representationStatusTag %}
+
+{% block pageHeading %}
+{% endblock %}
+
+
+{% block beforeContent %}
+	<div class="govuk-width-container">
+		<a class="govuk-back-link" href="{{ backLink }}">Back</a>
```

Comment:
backLinkUrl can be passed into the render and this will render from the extended template.